### PR TITLE
Review the digitizing "Rotate feature(s)" tool

### PR DESCRIPTION
**Pull request by DelazJ**
### Description

Goal: Add details and steps to https://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/editing_geometry_attributes.html#rotate-feature-s

Ticket: #

- [x] Backport to LTR documentation is required

### Minimal requirements for merging *(for maintainers)*

- [ ] The description of this PR is coherent with the manual and does not provide wrong information.
- [ ] This PR passes the checks.

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 1]
```diff
@@ -787,21 +787,29 @@ Likewise, you can create a translated copy of the feature(s) using the |moveFeat
...
+   opens showing the current :guilabel:`Rotation` angle. 
+#. Click on the map canvas when you are satisfied with the new placement or
+   manually enter the rotation angle in the text box. You can also use the
+   :guilabel:`Snap to °` box to constrain the rotation values.
+#. If you want to rotate several features at once, they shall be selected first,
+   and the rotation is by default processed around the centroid of their combined
+   geometries.
+
+You can also use an anchor point different from the default feature centroid:
+press :kbd:`Ctrl` button, click on the map canvas and that point will be
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]:
```suggestion
press the :kbd:`Ctrl` button, click on the map canvas and that point will be
```

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 2]
```diff
@@ -787,21 +787,29 @@ Likewise, you can create a translated copy of the feature(s) using the |moveFeat
...
+used as the new rotation center.
 
 If you hold :kbd:`Shift` before clicking on the map, the rotation will be done
 in 45 degree steps, which can be modified afterwards in the user input widget.
 
-To abort feature rotation, you need to click on |rotateFeature| :sup:`Rotate
-Feature(s)` icon.
+To abort feature rotation, press :kbd:`ESC` button or click on the |rotateFeature|
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]:
```suggestion
To abort feature rotation, press the :kbd:`ESC` button or click on the |rotateFeature|
```

File: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst [hunk 3]
```diff
@@ -787,21 +787,29 @@ Likewise, you can create a translated copy of the feature(s) using the |moveFeat
...
+
+#. Press the |rotateFeature| :sup:`Rotate Feature(s)` icon
+#. Then click on the feature to rotate. The feature's centroid is referenced as
+   rotation center, a preview of the rotated feature is displayed and a widget
+   opens showing the current :guilabel:`Rotation` angle. 
+#. Click on the map canvas when you are satisfied with the new placement or
+   manually enter the rotation angle in the text box. You can also use the
+   :guilabel:`Snap to °` box to constrain the rotation values.
+#. If you want to rotate several features at once, they shall be selected first,
+   and the rotation is by default processed around the centroid of their combined
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst]:
```suggestion
   and the rotation is by default around the centroid of their combined
```

**Comment by DelazJ** [comment]
Thanks @havatv

**Pull request merged**